README.md

```diff
@@ -1,14 +1,9 @@
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
+[![CI Status](https://github.com/AY2324S2-CS2103T-W09-3/tp/workflows/Java%20CI/badge.svg)](https://github.com/AY2324S2-CS2103T-W09-3/tp/actions)
 
 ![Ui](docs/images/Ui.png)
 
-* This is **a sample project for Software Engineering (SE) students**.<br>
-  Example usages:
-  * as a starting point of a course project (as opposed to writing everything from scratch)
-  * as a case study
-* The project simulates an ongoing software project for a desktop application (called _AddressBook_) used for managing contact details.
+* This is **a project for Software Engineering (SE) students based on the AddressBook-Level3 project created by the [SE-EDU initiative]**(https://se-education.org).<br>
+* The project simulates an ongoing software project for a desktop application (called _CulinaryContacts_) specially designed for family-owned restaurant owners or managers to manage contact details.
   * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
   * It comes with a **reasonable level of user and developer documentation**.
-* It is named `AddressBook Level 3` (`AB3` for short) because it was initially created as a part of a series of `AddressBook` projects (`Level 1`, `Level 2`, `Level 3` ...).
-* For the detailed documentation of this project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**.
-* This project is a **part of the se-education.org** initiative. If you would like to contribute code to this project, see [se-education.org](https://se-education.org#https://se-education.org/#contributing) for more info.
+* For the detailed documentation of this project, see the **[CulinaryContacts Product Website](https://nus-cs2103-ay2324s2.github.io/tp/)**.
```

Comment:
I think the website should be: https://ay2324s2-cs2103t-w09-3.github.io/tp/

Reply:
Thanks, changed to this.